### PR TITLE
Bump version to 4.6.1

### PR DESCRIPTION
**Pull request by nkolev92**
//cc
@rrelyea

### CODE REVIEW

File: .teamcity.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Collection of properties customizing TeamCity build configurations
 ReleaseLabel = rtm
-ReleaseProductVersion = 4.6.0
+ReleaseProductVersion = 4.6.1
```
Comment (rohit21agrawal) [inline .teamcity.properties:3]: Can you delete this file instead of updating it for no reason every time?

**Review by jainaashish — APPROVED** [review]

File: build/config.props [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <IsEscrowMode>true</IsEscrowMode>
     <MajorNuGetVersion Condition="'$(MajorNuGetVersion)' == ''">4</MajorNuGetVersion>
     <MinorNuGetVersion Condition="'$(MinorNuGetVersion)' == ''">6</MinorNuGetVersion>
-    <PatchNuGetVersion Condition="'$(PatchNuGetVersion)' == ''">0</PatchNuGetVersion>
+    <PatchNuGetVersion Condition="'$(PatchNuGetVersion)' == ''">1</PatchNuGetVersion>
```
Comment (rohit21agrawal) [inline build/config.props:8]: Could you also check if the target VS branch needs to change for 4.6.1?

File: .teamcity.properties (hunk 1 above)
Comment (nkolev92) [reply .teamcity.properties:3]: It's the 4.6.1 branch. 

Don't really want to do it here. 

If you think it needs done, let's do it in dev, not here.

File: build/config.props (hunk 2 above)
Comment (nkolev92) [reply build/config.props:8]: @PatoBeltran's gonna figure that out, as he's on hot seat. 
We're not sure. 

The idea is create the branch, merge the 3 fixes that need merging first. 

Have Pato do the insertion later.

File: .teamcity.properties (hunk 1 above)
Comment (rohit21agrawal) [reply .teamcity.properties:3]: Then don’t make this change either. It’s useless and unnecessary

Comment (rohit21agrawal) [reply .teamcity.properties:3]: Then don’t make this change either. It’s useless and unnecessary

Comment (nkolev92) [reply .teamcity.properties:3]: I really just looked at an old bump of the version, and just did the same logic. 
It's a temp branch that's only going to be used for these hotfixes. I'm not too concerned with how clean it is :)

File: build/config.props (hunk 2 above)
Comment (rohit21agrawal) [reply build/config.props:8]: my point was that he shouldn't have to generate another build to run with the insertion

File: .teamcity.properties (hunk 1 above)
Comment (rohit21agrawal) [reply .teamcity.properties:3]: which is why cleaner it is , the better :P

**Pull request merged**